### PR TITLE
#1266 Update the user guide for the common network problems

### PR DESCRIPTION
**Pull request by louie5486**
required by #1266

### Checklist
- [ ] [Review Contribution Guidelines](https://github.com/oharastream/ohara/blob/master/CONTRIBUTING.md)
- [ ] Add Reviewers (see [Authors](https://github.com/oharastream/ohara#authors))
- [ ] Add Assignees (of course, you are the assignee by default!)
- [ ] DON'T set a Milestone
- [ ] DON'T add any labels (PR labels will be added automatically by Jenkins)
- [ ] Link to an issue(You can do so by using the required by keyword: required by #issue_number)
- [ ] Add unit tests (or please explain why this PR is not worth having new tests)
- [ ] Pass QA (Sometimes you may encounter an existing flaky test which would obstruct you from getting QA passed. If the failed tests are unrelated, you can add a comment in the PR thread)

### CODE REVIEW

File: docs/user_guide.md [hunk 1]
```diff
@@ -72,6 +72,13 @@ again) don't happen on your nodes.
 After validating your network configurations layer by layer, you could try filing issue on github if you still can't
 get Ohara to work. 
 
+###  We often encounter problems with network problems.
+After install Docker-ce package in Centos,the network default policy is block docker's bridge to host network,
+You must open a rule on the firewall:
+
+```
```
Comment (chia7712) [inline docs/user_guide.md]:
```bash

File: docs/user_guide.md [hunk 2]
```diff
@@ -72,6 +72,13 @@ again) don't happen on your nodes.
 After validating your network configurations layer by layer, you could try filing issue on github if you still can't
 get Ohara to work. 
 
+###  We often encounter problems with network problems.
+After install Docker-ce package in Centos,the network default policy is block docker's bridge to host network,
```
Comment (vitojeng) [inline docs/user_guide.md]: nit:
Centos --> CentOS

File: docs/user_guide.md (hunk 1 above)
Comment (louie5486) [reply docs/user_guide.md]: ok,got it!!

File: docs/user_guide.md [hunk 3]
```diff
@@ -72,6 +72,13 @@ again) don't happen on your nodes.
...
+You must open a rule on the firewall:
```
Comment (vitojeng) [inline docs/user_guide.md]: must --> **must**

IMO, highlight it would be better... 😆

File: docs/user_guide.md (hunk 2 above)
Comment (louie5486) [reply docs/user_guide.md]: yes

**Comment by vitojeng** [comment]
@louie5486 Follow the checklist.
You need to append `required by #1266` on top of the PR description. And then Github will create a link to #1266 automatically for you.

**Comment by vitojeng** [comment]
LGTM, +1
Thank you for the PR, well done!

**Pull request merged**